src/components/subscriptions/SubscriptionExpirationModal.jsx

```diff
@@ -12,14 +12,13 @@ import {
   SUBSCRIPTION_DAYS_REMAINING_EXCEPTIONAL,
   SEEN_SUBSCRIPTION_EXPIRATION_MODAL_COOKIE_PREFIX,
 } from './constants';
-
-import Img from '../../components/Img';
-import edxLogo from '../../images/edx-logo.png';
+import { configuration } from '../../config';
+import Img from '../Img';
 import { formatTimestamp } from '../../utils';
 
 export const MODAL_DIALOG_CLASS_NAME = 'subscription-expiration';
 
-
+const edxLogo = configuration.LOGO_URL;
```

Comment:
nit: Should this be defined where the variable is used in `renderExpiredBody()`?

Reply:
Yeah, I think this was from my initial pass when I was just seeing what worked and not really paying attention to how the variable was being used. I'll see about cleaning it up.